service/static/js/index.js

```diff
@@ -324,4 +324,100 @@ $(function () {
     });
   });
 
+  // ****************************************
+  // Delete a Product
+  // ****************************************
+
+  $("#delete-product").click(function (event) {
+    event.preventDefault();
+    var product_id = $("#wishlist-product-id").val();
+    var wishlist_id = $("#wishlist-id").val();
+
+    var ajax = $.ajax({
+      type: "DELETE",
+      url: `/wishlists/` + wishlist_id + `/products/` + product_id,
+      contentType: "application/json",
+      data: '',
+    })
+
+    ajax.done(function(res){
+      clear_form_data();
+      flash_message("Success");
+    });
+
+    ajax.fail(function(res){
+      flash_message("Product doesn't exist!");
```

Comment:
Should this be `res.responseJSON.message` like everywhere else?

Reply:
Yes, I will change that.